iron-component-page.html

```diff
@@ -198,40 +205,52 @@
     <hydrolysis-analyzer id="analyzer" src="[[_srcUrl]]" transitive="[[transitive]]" clean analyzer="{{_hydroDesc}}" loading="{{_hydroLoading}}"></hydrolysis-analyzer>
     <iron-ajax id="ajax" url="[[docSrc]]" handle-as="json" on-response="_handleAjaxResponse" on-error="_handleError"></iron-ajax>
 
-    <paper-header-panel id="headerPanel" mode="[[scrollMode]]">
-      <paper-toolbar catalog-hidden>
-        <div class="docs-header">
-          <!-- TODO: Replace with paper-dropdown-menu when available -->
-          <select id="active" value="[[active]]" on-change="_handleMenuItemSelected">
-            <template is="dom-repeat" items="[[docElements]]">
-              <option value="[[item.is]]">[[item.is]]</option>
-            </template>
-            <template is="dom-repeat" items="[[docBehaviors]]">
-              <option value="[[item.is]]">[[item.is]]</option>
-            </template>
-          </select>
-        </div>
-        <iron-selector attr-for-selected="view" selected="{{view}}" id="links" hidden$="[[!docDemos.length]]">
-          <a view="docs"><iron-icon icon="description"></iron-icon> Docs</a>
-          <a view="[[_demoView(docDemos.0.path)]]"><iron-icon icon="visibility"></iron-icon> <span>Demo</span></a>
-        </iron-selector>
-      </paper-toolbar>
-      <div id="content">
-        <iron-selector id="view" selected="[[_viewType(view)]]" attr-for-selected="id">
-          <div id="docs">
-            <div id="catalog-heading" catalog-only>
-              <h2><span>[[active]]</span> <span class="version" hidden$="[[!version]]">[[version]]</span></h2>
-            </div>
-            <iron-doc-viewer prefix="[[_fragmentPrefix]]" id="viewer" descriptor="{{_activeDescriptor}}"
-              on-iron-doc-viewer-component-selected="_handleComponentSelectedEvent"></iron-doc-viewer>
-            <div id="nodocs" hidden$="[[_activeDescriptor]]" class="no-docs">
-              No documentation found.
-            </div>
-          </div>
-          <div id="demo"></div>
+    <paper-drawer-panel>
+      <div drawer catalog-hidden>
```

Comment:
We need to make the drawer scrollable (overflow: auto) since the list could be long.  e.g. check the elements list in `neon-animation` component.